src/inmanta/ast/statements/generator.py

```diff
@@ -696,7 +696,7 @@ def requires_emit(
         # Schedule execution to resume when the condition can be executed
         resumer: RawResumer = ConditionalExpressionResumer(self, result, lhs=lhs)
         self.copy_location(resumer)
-        RawUnit(queue, resolver, self.condition.requires_emit(resolver, queue), resumer)
+        RawUnit(queue, resolver, self.condition.requires_emit(resolver, queue), resumer, False)
```

Comment:
```suggestion
        RawUnit(queue, resolver, self.condition.requires_emit(resolver, queue), resumer, override_exception_location=False)
```
Just a tad more readable. I should probably have made it a kw-only arg in the first place.